src/utils/getSocialIdentitiesIcon.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { IdentityTypes } from "@lib/@types/Identity";
+import { AppleIcon } from "@src/components/display/fewlines/Icons/AppleIcon/AppleIcon";
+import { FacebookIcon } from "@src/components/display/fewlines/Icons/FacebookIcon/FacebookIcon";
+import { GithubIcon } from "@src/components/display/fewlines/Icons/GithubIcon/GithubIcon";
+import { GoogleIcon } from "@src/components/display/fewlines/Icons/GoogleIcon/GoogleIcon";
+import { KakaoTalkIcon } from "@src/components/display/fewlines/Icons/KakaoTalkIcon/KakaoTalkIcon";
+import { LineIcon } from "@src/components/display/fewlines/Icons/LineIcon/LineIcon";
+import { MicrosoftIcon } from "@src/components/display/fewlines/Icons/MicrosoftIcon/MicrosoftIcon";
+import { NaverIcon } from "@src/components/display/fewlines/Icons/NaverIcon/NaverIcon";
+import { PaypalIcon } from "@src/components/display/fewlines/Icons/PaypalIcon/PaypalIcon";
+import { StravaIcon } from "@src/components/display/fewlines/Icons/StravaIcon/StravaIcon";
+import { VKontakteIcon } from "@src/components/display/fewlines/Icons/VKontakteIcon/VKontakteIcon";
+
+const SOCIAL_IDENTITIES_ICONS = {
+  APPLE: <AppleIcon />,
+  FACEBOOK: <FacebookIcon />,
+  GITHUB: <GithubIcon />,
+  GOOGLE: <GoogleIcon />,
+  KAKAO_TALK: <KakaoTalkIcon />,
+  LINE: <LineIcon />,
+  NAVER: <NaverIcon />,
+  PAYPAL: <PaypalIcon />,
+  STRAVA: <StravaIcon />,
+  VKONTAKTE: <VKontakteIcon />,
+  MICROSOFT: <MicrosoftIcon />,
+};
```

Comment:
You should pull master, because you are missing `DECATHLON` :).

Reply:
Ok fixed 61bf568, I added the icon manually since I transferred this object to a new file, a rebase won't fix the problem